packages/utils/src/fallbackSubgraph.ts

```diff
@@ -36,85 +48,127 @@ type Subgraph<T extends SubgraphSdk> = {
   };
 };
 
+// Return types for subgraph health endpoint.
+type SubgraphHealthError = {
+  message: string;
+  block: number;
+  handler: any;
+};
+
+type SubgraphHealth = {
+  data: {
+    // TODO: Deprecate this unnecessary wrapping object.
+    indexingStatusForCurrentVersion: {
```

Comment:
If we are releasing a breaking change, and this isn't live yet, why not just deprecate now?

Reply:
Oh is this nested in a `data.data.indexingStatusForCurrentVersion`? Let's just fix this now